ht-3/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <style>
+    .planets-table,
+    .planets-table th,
+    .planets-table td {
+      border: 3px solid black;
+    }
+    .planets-table {
+      border-collapse: collapse;
+    }
+    .planets-table-td {
```

Comment:
let's rename to `planets-table-selected-td` or `planets-table-td--selected` if you use BEM in this project